### DISCUSSION TITLE
以前编辑器配置中的markedRenderer去哪里了

### DISCUSSION THREAD

**Discussion by daidaibg** [imzbf/md-editor-v3#281] [Q&A]
![image](https://github.com/imzbf/md-editor-v3/assets/55817318/bd25e020-c7fb-4a12-902d-c3eb58749963)
新版本要怎么去改呢 谢谢

**Comment by daidaibg** [comment]
标题ID  新版本是不是只需要绑定这个属性
以前a标签的属性现在要怎么修改呢 
![image](https://github.com/imzbf/md-editor-v3/assets/55817318/80bb4931-bca4-4a6b-9f09-2d28329172b1)

**Reply by imzbf** [reply]
> 标题ID 新版本是不是只需要绑定这个属性 以前a标签的属性现在要怎么修改呢 ![image](https://user-images.githubusercontent.com/55817318/242505692-80bb4931-bca4-4a6b-9f09-2d28329172b1.png)


修改`a`标签参考下(#274 

现在编译库是`markdown-it`，扩展是根据它来的哈

**Reply by daidaibg** [reply]
Great, thanks!

**Comment by daidaibg** [comment]
Great, thanks!
